nuitka/Options.py

```diff
@@ -2067,51 +2067,40 @@ def isMacOSUiElementApp():
 
 def _getPythonFlags():
     """*list*, values of ``--python-flag``"""
-    # There is many flags, pylint: disable=too-many-branches
-
-    # singleton, pylint: disable=global-statement
     global _python_flags
 
     if _python_flags is None:
         _python_flags = set()
 
+        flag_dict = {
```

Comment:
I would of course define this on the global level, but sure enough, very nice.

Reply:
Actually I retract my statement, using this dictionary locally is good, as it's a single time use only